server/api/home.go

```diff
@@ -32,3 +32,7 @@ func newHomeHandler(rd *render.Render) *homeHandler {
 func (h *homeHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.rd.HTML(w, http.StatusOK, "index", r.Host)
 }
+
+func (h *homeHandler) Ping(w http.ResponseWriter, r *http.Request) {
```

Comment:
What is this for?

I considered you might be interested in [Keep-Alive](https://www.w3.org/Protocols/HTTP/1.1/draft-ietf-http-v11-spec-01#Keep-Alive).